src/routes/chat/ChatHeader/ChatHeader.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import ChatCreateOffer from './ChatCreateOfferHeader';
+import ChatDefaultHeader from './ChatDefaultHeader';
+import ChatOfferDetailsHeader from './ChatOfferDetailsHeader';
+import './ChatHeader.scss';
+
+const ChatHeader = () => {
+  const [header, setHeader] = useState(0);
+  const userType = useSelector(state => state.misc.userType);
+  const chatRoom = useSelector(state => state.chat.chatRoom);
+
+  const getSellerBuyerBids = () => {
+    const seller = {
+      price: chatRoom.sellerPrice,
+      numberOfShares: chatRoom.sellerNumberOfShares
+    };
+    const buyer = {
+      price: chatRoom.buyerPrice,
+      numberOfShares: chatRoom.buyerNumberOfShares
+    };
+    return userType === 'seller'
```

Comment:
!!! Do not use userType. Check the id instead. A user can be both seller and buyer at the same time, and may be buying for one company, but selling for another. This is not accurate!